src/BloomExe/MiscUI/BloomIntegrityDialog.cs

```diff
@@ -1,18 +1,13 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
-using System.Diagnostics;
-using System.Drawing;
 using System.IO;
 using System.Linq;
+using System.Management;
```

Comment:
This may need #if !__MonoCS__.  (which may be why it was being used explicitly before)